cmd/restic/main.go

```diff
@@ -102,6 +103,30 @@ func tweakGoGC() {
 	}
 }
 
+func printExitError(code int, message string) {
+	if globalOptions.JSON {
+		type jsonExitError struct {
+			MessageType string `json:"message_type"` // exit_error
+			Code        int    `json:"code"`
+			Error       string `json:"error"`
```

Comment:
Do we want to mimic the backup/restore errors here with the struct shape from #4946 that holds a string? Or try to distance ourselves from that arguably accidental setup? Or... a half way measure and name this string field `message` to avoid comparisons to the `error` field in those JSON blobs?

Reply:
I prefer the field name `message`. That would then be consistent with an inlined `error` object. Also `jsonExitError.Code` and `jsonExitError.Message` read much better than ``jsonExitError.Error`.

Reply:
Done (switched to `message`).